certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
+                    "operationId": "GetProcessApprovalRequestStages",
+                    "itemValuePath": "id",
+                    "itemTitlePath": "name",
+                    "parameters": {
+                      "approval_process_request_id": {
+                        "parameterReference": "body/approval_process_request_id"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetProcessApprovalRequestStages",
+                    "value-path": "id",
+                    "value-title": "name",
+                    "value-collection": "msg",
+                    "parameters": {
+                      "approval_process_request_id": {
+                        "parameter": "approval_process_request_id"
+                      }
+                    }
+                  }
+                },
+                "callback_url": {
+                  "type": "string",
+                  "description": "callback_url",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "-"
+                }
+              },
+              "required": [
+                "approval_process_request_id",
+                "approval_stage_id",
+                "callback_url"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/get_approval_process_requests": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "Status"
+                },
+                "subject": {
+                  "type": "string",
+                  "description": "Subject"
+                },
+                "msg": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "integer",
+                        "description": "Request ID",
+                        "title": "",
+                        "format": "string"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "Request Date"
+                      }
+                    }
+                  },
+                  "description": "msg"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Approval Processes Requests",
+        "description": "Get Approval Process Requests",
+        "operationId": "GetApprovalProcessRequests",
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/get_approval_stages/{approval_process_request_id}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
Please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.